### PR TITLE
Added unique constraint on name of pipeline in database.

### PR DESCRIPTION
**Pull request by eea02**
Related to issue #252 or EDEM-138.

### CODE REVIEW

File: db/mysql/schema.sql [hunk 1]
```diff
@@ -438,6 +438,8 @@ ADD FOREIGN KEY (`user_id`)
     REFERENCES `usr_user` (`id`)
 	ON UPDATE CASCADE ON DELETE CASCADE;
 
+ALTER TABLE ppl_model ADD UNIQUE (name(255));
```
Comment (tomas-knap) [inline db/mysql/schema.sql]: The limit should be the same as the limit in cz.cuni.mff.xrg.odcs.commons.app.constants.LengthLimits.PIPELINE_NAME, which is enforced in GUI

File: db/postgresql/updates/1.6.0-update.sql [hunk 2]
```diff
@@ -1,4 +1,5 @@
-
 -- Update version.
 UPDATE "properties" SET "value" = '001.006.000' WHERE "key" = 'UV.Core.version';
 UPDATE "properties" SET "value" = '001.003.000' WHERE "key" = 'UV.Plugin-DevEnv.version';
+
+ALTER TABLE ppl_model ADD CONSTRAINT ppl_model_name_constraint UNIQUE(name);
```
Comment (tomas-knap) [inline db/postgresql/updates/1.6.0-update.sql]: Please add also update script for mysql

**Comment by tomas-knap** [comment]
What was the motivation for such quick partial fix? 

It  enforces uniqueness of pipeline names, but does not enforce uniqueness of other concepts, such as dpu template names.

File: db/mysql/schema.sql (hunk 1 above)
Comment (eea02) [inline db/mysql/schema.sql]: The problem is, that MySQL does not support such big indexes. We will either limit uniqueness on smaller size, or we will have to use innodb_large_prefix options. (http://mechanics.flite.com/blog/2014/07/29/using-innodb-large-prefix-to-avoid-error-1071/)

**Comment by eea02** [comment]
Request as per JIRA https://team.eea.sk/jira/browse/EDEM-138 was that Master API should reflect functionality in frontend.

**Comment by skrchnavy** [comment]
@tomas-knap there was no request for uniqueness of templates. There was discrepancy between frontend and API, frontend blocked creation of pipelines if such name used and API allowed it.

(for dpu templates naming we have issues like: #84, #177)

File: db/mysql/schema.sql (hunk 1 above)
Comment (tomas-knap) [inline db/mysql/schema.sql]: Ok, so we agreed, we will leave 255 there, but with a comment why 255 should be there and how this can be improved.

**Pull request merged**